next.config.js

```diff
@@ -1,3 +1,5 @@
+const withImages = require('next-images')
+module.exports = withImages()
```

Comment:
Semicolons are still highly encouraged for readability.

```suggestion
const withImages = require('next-images');
module.exports = withImages();
```